addons/website_slides/static/tests/tours/slides_tour_tools.js

```diff
@@ -87,6 +95,126 @@ var addArticleToSection = function (sectionName, pageName, backend) {
 }];
 };
 
+const fillInFileInput = async (input, name, type, content) => {
+    const blob = await fetch(`data:${type};base64,${content}`).then(res => res.blob());
+    const file = new File([blob], name, { type: type});
+    const dataTransfer = new DataTransfer();
+    dataTransfer.items.add(file);
+    input.files = dataTransfer.files;
+    input.dispatchEvent(new Event('change', { bubbles: true }));
+};
+
+const checkURLContent = async (url, name, type, expectedContent) => {
+	const blob = await fetch(url).then(res => res.blob());
```

Comment:
Indent is still strange by the way, seems like a lot more starting space than the previous function.